lib/AutocompleteInput.js

```diff
@@ -24,14 +25,17 @@ class AutocompleteInput extends Component {
     }
 
     render() {
-        const {custom = {}, ...props} = this.props;
-
+        const {containerStyle, inputStyle} = this.props;
         return (
-            <View style={[style.container, custom.container]}>
+            <View style={{...style.container, ...containerStyle}}>
                 <TextInput
-                    {...props}
-                    onChangeText={this._handleChangeText}
-                    style={[style.input, custom.input]} />
+                  value={this.props.value}
+                  placeholder={this.props.placeholder}
+                  debounce={this.props.debounce}
+                  onChangeText={this._handleChangeText}
+                  selectionColor={this.props.selectionColor}
+                  onChangeTextSettle={this.props.onChangeTextSettle}
```

Comment:
The `onChangeTextSettle` prop doesn't need to be passed into `TextInput`, it's a custom event for the parent of `AutocompleteInput` to listen to.